app/Services/Sftp.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Services;
+
+use App\Services\BaseService;
+use App\Services\Category;
+use App\Shell\Docker;
+use App\Shell\Environment;
+use App\Shell\Shell;
+
+class Sftp extends BaseService
+{
+    protected static $category = Category::STORAGE;
+
+    protected $organization = 'atmoz';
+    protected $imageName = 'sftp';
+    protected $defaultPort = 22;
+    protected $tag = 'alpine';
+    protected $prompts = [
+        [
+            'shortname' => 'user_name',
+            'prompt' => 'What will the default username be?',
+            'default' => 'foo',
+        ],
+        [
+            'shortname' => 'password',
+            'prompt' => 'What will the default password be?',
+            'default' => 'pass',
+        ],
+        [
+            'shortname' => 'upload_directory',
+            'prompt' => 'Where will files be uploaded?',
+            'default' => 'upload',
+        ],
+        [
+            'shortname' => 'mapped_directory',
+            'prompt' => 'Which local directory should be mapped inside? (nothing if null)',
+            'default' => '',
+        ],
+    ];
+
+    protected $dockerRunTemplate = '-p "${:port}":22 \
+        "${:organization}"/"${:image_name}":"${:tag}" \
+        "${:user_config}"';
+
+    protected static $displayName = 'SFTP';
+
+    public function __construct(Shell $shell, Environment $environment, Docker $docker) {
+        parent::__construct($shell, $environment, $docker);
+
+        $this->defaultPrompts = array_map(function ($prompt) {
+            if ($prompt['shortname'] === 'tag') {
+                $prompt['default'] = $this->tag;
+            }
+
+            return $prompt;
+        }, $this->defaultPrompts);
+    }
+
+    protected function prompts(): void {
+        parent::prompts();
+
+        if($this->promptResponses['mapped_directory'] !== "") {
```

Comment:
There are a few lines in here that I know don't work with the Tlint specs. Would you mind running TLint on this PR? Thanks!

https://github.com/tighten/tlint

Reply:
I just ran the linter and it just spits out **LGTM!**, which looks like there are no lints or it was successful?

Reply:
I fixed the issue withe position of the opening curly bracket.

Reply:
Hm. LGTM = Looks good to me, but this is definitely not tlint style. Let me try it.

Reply:
@chrisdicarlo Ahh, I mixed up. We *do* want to run Tlint, but this particular set of issues comes from the PSR's, so that'd come from running PHPCS. You can learn more in our contributing.md: https://github.com/tighten/takeout/blob/main/CONTRIBUTING.md

Reply:
My bad, I forgot to run it through phpcbf before I submitted the PR.  I just pushed a commit to fix things up.